author.tf

```diff
@@ -354,6 +363,16 @@ module "author-api" {
 {
   "Version": "2012-10-17",
   "Statement": [
+      {
+          "Sid": "",
+          "Effect": "Allow",
+          "Action": [
+              "s3:ListObjects",
+              "s3:ListBucket",
+              "s3:GetObject"
+          ],
+          "Resource": "arn:aws:s3:::*"
```

Comment:
I would use `${var.author_secrets_bucket_name}` to restrict access to only the bucket required